samples/LargeResponseApp/Startup.cs

```diff
@@ -34,5 +35,17 @@ public void Configure(IApplicationBuilder app)
                 }
             });
         }
+
+        public static void Main(string[] args)
+        {
+            var config = WebApplicationConfiguration.GetDefault(args);
+
+            var application = new WebApplicationBuilder()
+                .UseConfiguration(config)
```

Comment:
Or just inline it:
`.UseConfiguration(WebApplicationConfiguration.GetDefault(args))`